lib/slack-ruby-bot/commands/base.rb

```diff
@@ -122,6 +131,23 @@ def finalize_routes!
           command command_name_from_class
         end
 
+        def match_attachments(data, route)
+          fields_to_scan = %i[pretext text title]
+          data.attachments_.each do |attachment|
```

Comment:
Unnecessary usage of `under-bang` from `Hashie::Mash` as there is a validation against missing data (`data_missing?`). Will take it out.